.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+### Pull reuqest for @Cloudianry/Base
+
+
+#### What does this PR solve?
+... A few words
+
+
+#### Final checklist
+[ ] JSdoc - Add proper docstrings based on our PHP implementation
+[ ] JSdoc - Hide private functions using @private
+[ ] Implementation is aligned with PHP Reference(If different, please specify why)
+
```

Comment:
Tests?

Reply:
Since this is still only developed internally I didn't find a need to add it in the PR template,
adding that we have automatic tools that detect code coverage issues.

Reply:
I think it needs to be there. Even if on some PRs you won't "check it"

Reply:
Alright, Done. Merging